grants_tagger/bertmesh/results.json

```diff
@@ -0,0 +1 @@
+{"p": 0.6955707011887685, "r": 0.569327457868485, "f1": 0.626149221959104, "th": 0.5}
```

Comment:
It'd be nice to have train information such as time to train if I want to reproduce.

Reply:
Sure it can be added, I was not using our main train script to accelerate experimentation but I can add it easily.